### ISSUE TITLE
Change packaging to nodejs toolchain

### ISSUE THREAD

**Issue by mach25** [Atmosphere/atmosphere-javascript#200]
I'm wondering if you are interested in packaging atmosphere-javascript with nodejs and npm instead of java and maven?

I have made a first attempt at restructuring the project here https://github.com/mach25/atmosphere-javascript/tree/npm

I made a fast conversion to es6 with babel transpiling, linting with eslint standard profile and writing a unit test with mocha and chai. (Also fixed a bug i found in my old pull request with a unit test)

What is needed is packaging and deployment.

**Comment by jfarcand** [comment]
@mach25 +1 Extremely interested by the  end result :-)

**Comment by mach25** [comment]
Ok, so now i have webpack bundling for packaging for deployment on cdn.
I don't have any experience with bower but i created a bower.json.

I have it published on my user on npmjs https://www.npmjs.com/package/@mach25/atmosphere-javascript

I did a small test and it seemed to be working when using npm to depend on it.

I also created a travis build to test out the mocha tests https://travis-ci.org/mach25/atmosphere-javascript

Anything else that needs to be looked at?
